maintenance/maintenance.html

```diff
@@ -4,17 +4,17 @@
 <html lang="en-US">
     <head>
         <meta charset="utf-8">
-        <title>Donate to Thunderbird | Give to Thunderbird</title>
+        <title>Give to Thunderbird</title>
```

Comment:
Is this maintenance page only for Thunderbird site though?

Reply:
If not we should open a follow-up ticket so we can have a Foundation version of maintenance page.

Reply:
```suggestion
        <title>Mozilla Giving Platform</title>
```